spec/vcr/vcr_setup.rb

```diff
@@ -8,9 +8,3 @@
   c.debug_logger = File.open('spec/vcr.log', 'w')
   c.configure_rspec_metadata!
 end
-
-RSpec.configure do |c|
-  # so we can use :vcr rather than :vcr => true;
-  # in RSpec 3 this will no longer be necessary.
```

Comment:
I was getting deprecation warning every time I run rspec. This is related https://git.io/vavzJ